jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/OperationsEventIT.java

```diff
@@ -282,7 +285,7 @@ void testGetAllEventsByTransportCallIDWithEventCreatedDateTimeRange() {
         .statusCode(200)
         .contentType(ContentType.JSON)
         // validate an exact match.
-        .body("size()", equalTo(1))
+        .body("size()", greaterThanOrEqualTo(1))
```

Comment:
The comment suggests this should be an exact match. I would prefer we kept `equalTo` and just bumped the exact match from `1` to whatever the new number is.

Reply:
Alternatively, tighten the parameters to reduce it to the exact match it was.

Reply:
This is mostly done to avoid it failing in case a similar operations event is added to test data. Will revert it to 1